packages/core/src/actor/instance.ts

```diff
@@ -384,7 +396,9 @@ export class ActorInstance<
 				});
 
 				// Look up function
-				const fn: unknown = this.#config.actions[event.a];
+				const fn: unknown = event.i
```

Comment:
this api makes it difficult to follow what's actually going to be called.

instead, we should have an enum for what's going to be called. either an action or an internal fucntion. the internal fucntion should be hardcoded as an enum (ie only "checkConnectionLiveness") instead of trying to dynamically look it up.